src/components/05_pages/NodeForm/index.js

```diff
@@ -67,11 +62,12 @@ class NodeForm extends React.Component {
   };
 
   componentDidMount() {
-    this.props.requestUser(1);
-    this.props.requestSchema({
-      entityTypeId: this.props.entityTypeId,
-      bundle: this.props.bundle,
-    });
+    if (!this.props.schema) {
+      this.props.requestSchema({
```

Comment:
Should we at least add a todo to remove this? It would be nice if this would be consistently done on the parent component.

Reply:
Good question. I'm happy with either way. Having less data fetching inside NodeForm itself is certainly not a bad idea

Reply:
I think this direction makes even more sense in the long run when we move towards supporting other entity types 🙂

Reply:
This will be somewhat difficult to achieve. We don't currently capture the bundle in the route, just the `NID`. You could compute the bundle by looking at the `type` of the entity, however that's sort of fragile.

Reply:
Agreed that this should happen in a follow-up! Let's open the issue and add a todo comment. 👍